tests/calc/utilities/poscar/test_modify_vacuum.py

```diff
@@ -9,25 +9,29 @@
 
 import numpy as np
 import pytest
-import pytest_cases
+from pytest_cases import parametrize
+from pytest_cases import parametrize_with_cases
 
-from viperleed.utilities.poscar.modify_vacuum import ModifyVacuumCLI
+from viperleed.calc.classes.slab.errors import VacuumError
 from viperleed.calc.classes.slab.surface_slab import _MIN_VACUUM
+from viperleed.utilities.poscar import modify_vacuum
```

Comment:
```suggestion
from viperleed.utilities.poscar.modify_vacuum import modify_vacuum
```

Perhaps nicer, it avoids the double `modify_vacuum.modify_vacuum`. If you go this way, also re-sort the imports so this comes after exceptions (capitals come earlier in ASCII).